services/movie.py

```diff
@@ -14,20 +16,23 @@
 
     if actors_ids:
         queryset = queryset.filter(actors__id__in=actors_ids)
-
+    if title:
+        queryset = queryset.filter(title__icontains=title)
     return queryset
 
 
 def get_movie_by_id(movie_id: int) -> Movie:
     return Movie.objects.get(id=movie_id)
```

Comment:
Consider adding error handling for the `get_movie_by_id` function to manage cases where a movie with the specified ID does not exist, such as using `Movie.objects.filter(id=movie_id).first()` or handling `Movie.DoesNotExist` exception.